app/views/admin/editions/_show_document_metadata.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="page-header">
+  <h1><%= @edition.title %></h1>
+  <p class="lead"><%= @edition.summary %></p>
+  <dl class="clearfix dl-horizontal">
+    <dt>Type of document:</dt>
+    <dd><%= edition_type(@edition) %></dd>
+    <dt>Status:</dt>
+    <dd>
+      <% status_text = [@edition.state.capitalize]
+         # TODO: remove unpublishing information once we have a separate state for unpublished editions
+         status_text << "unpublished #{time_ago_in_words(@edition.unpublishing.created_at)} ago" if @edition.unpublishing.present? %>
+      <%= status_text.join(', ') %>
+    </dd>
+    <dt>Change type:</dt>
+    <dd><%= @edition.minor_change? ? 'Minor' : 'Major' %></dd>
+    <% if @edition.respond_to?(:organisations) %>
+      <dt>Organisations:</dt>
+      <dd><%= joined_list(@edition.organisations.map { |o| o.name }) %></dd>
+    <% end %>
+  </dl>
```

Comment:
Sorry, I missed this when the design was going on. 

I don't think we want to show status or change_type since that would reflect data that Content Publisher is now authoritative on. 

I'd suggest rather than making this partial it'd also be slightly cleaner to have this distinct (despite the code duplication) as we'd anticipate that any changes to this metadata for live editions probably is not what locked documents needs so could cause issues. So I think it'd be best to copy and paste the the title, summary, document and organisations parts into the locked view. I figure that should come out at about ~10 lines of code so probably not a painful amount of duplication.

Reply:
I've removed the partial and added only the necessary metadata to the `show_locked` view.